pkg/web/apps/apps_psql_repository.go

```diff
@@ -193,7 +193,7 @@ func (a *appsPostgreSQLRepository) GetDeviceByVersionAndAppID(version string, ap
 func (a *appsPostgreSQLRepository) GetAppByAppID(appID string) (*models.App, error) {
 	app := models.App{}
 
-	sqlStatement := `SELECT id,LOWER(app_id),app_name,deleted_at FROM app WHERE LOWER(app_id)=$1;`
+	sqlStatement := `SELECT id,app_id,app_name,deleted_at FROM app WHERE LOWER(app_id)=$1;`
 
```

Comment:
I understand that is missing this place: https://github.com/aerogear/mobile-security-service/blob/57e14c8d31ee55843423f09615e1b88bc3972927/pkg/web/apps/apps_psql_repository.go#L112

Reply:
Thank you :smile:

Reply:
After investigating further, the line you quoted is getting an app by its ID (PK, uuid), not the app ID (string).

Reply:
I think you intended to reference line 227:

https://github.com/aerogear/mobile-security-service/blob/57e14c8d31ee55843423f09615e1b88bc3972927/pkg/web/apps/apps_psql_repository.go#L217

Reply:
Hi @craicoverflow has the GetActiveAppID which is by the appID. Could you please check this one? It is that one which I am saying that is missing this change.

Reply:
Yep I am updating this now.